recipes/mender-client/debian-master/control

```diff
@@ -20,19 +22,22 @@ Package: mender-update
 Conflicts: mender
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
+# debian buster Depends: libarchive13 (>= 3.3.3), libc6 (>= 2.28), libdbus-1-3 (>= 1.9.14), libgcc-s1 (>= 3.5), liblmdb0 (>= 0.9.7), libssl1.1 (>= 1.1.1), libstdc++6 (>= 9)
 Description: Mender update
  Mender-update is an open source over-the-air (OTA) software updater for embedded Linux devices.
 
 Package: mender-auth
 Conflicts: mender
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
+# debian buster Depends: libc6 (>= 2.28), libdbus-1-3 (>= 1.9.14), libgcc-s1 (>= 3.5), libssl1.1 (>= 1.1.1), libstdc++6 (>= 9)
 Description: Mender auth
  Mender-auth enables centralized authentication against a Mender server.
 
 Package: mender-client-dev
 Architecture: all
 Depends: mender-client (= ${binary:Version}), ${misc:Depends}
+# debian buster Depends: mender-client (= ${binary:Version})
```

Comment:
`#` is part of the syntax?? Really weird...

Reply:
No, this is our hack to hard-code dependencies and not invoke Debian Helpers. See [here](https://github.com/mendersoftware/mender-dist-packages/blob/f68ac9acf101e0dd9803fe4f7498b41155975315/mender-deb-package#L157).

It is what we have been using for all recipes until the C++ client introduction when we _tried_ to be more streamlined.

Reply:
Ah, this is our thing, got it.